symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkConfig.java

```diff
@@ -17,4 +17,7 @@
     private BdkExtAppConfig app = new BdkExtAppConfig();
     private BdkSslConfig ssl = new BdkSslConfig();
 
+    private BdkRetryConfig retry = new BdkRetryConfig();
```

Comment:
we have it at the same level here and also part of BdkDatafeedConfig, is it normal?

Reply:
The idea of Darlys is that we have a global retry configuration for all services in the bot, but in each service configuration, we can override this retry config.

Reply:
we need to document that in the readme file, to explain to developer community this approach!!!